packages/language-service/src/completions.ts

```diff
@@ -28,6 +28,8 @@ const hiddenHtmlElements = {
   link: true,
 };
 
+const ANGULAR_PSEUDO_ELEMENTS = ['ng-container', 'ng-content', 'ng-template'] as const ;
```

Comment:
Maybe this would be better as

```suggestion
const ANGULAR_PSEUDO_ELEMENTS: ReadonlyArray<string> = ['ng-container', 'ng-content', 'ng-template'];
```

?